test/__tests__/data/customerio_input.json

```diff
@@ -4153,5 +4153,115 @@
       "Transformations": [],
       "IsProcessorEnabled": true
     }
-  }
+  },
+    {
+      "description": "successful group call with userId and groupId as an integer",
+      "message": {
+        "type": "group",
+        "header": {
+          "content-type": "application/json; charset=utf-8"
+        },
+        "sentAt": "2023-03-28T09:36:49.882Z",
+        "traits": {
+          "city": "Frankfurt",
+          "name": "On-Apply GmbH 2",
+          "state": "Hessen",
+          "isFake": true,
+          "address": "Solmsstraße 83",
+          "country": "DE",
+          "website": "http://www.onapply.de",
```

Comment:
please change these to something generic

Reply:
updated